src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,119 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+/**
+ * Deals with loading tasks from the file and saving tasks in the file.
+ */
+public class Storage {
+    /** The file where tasks are loaded from and saved to. */
+    private File file;
+
+    /**
+     * Creates a new Storage that loads tasks from and saves tasks to the given file.
+     * @param filePath The file path of the file.
+     */
```

Comment:
This comment is missing the empty line between description and parameter section as stated in the coding convention page. Happens a few times in this class, so I'll just comment once here!

Reply:
Noted!